html/semantics/scripting-1/the-template-element/template-element/template-content-node-document-document-properties.sub.html

```diff
@@ -0,0 +1,41 @@
+<meta charset=windows-1252>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+  test(() => {
+    assert_equals(document.origin, "{{location[scheme]}}://{{location[host]}}")
+    var t = document.createElement("template")
+    assert_equals(t.content.ownerDocument.origin, "{{location[scheme]}}://{{location[host]}}")
+  }, "document.origin")
+
+  test(() => {
+    assert_equals(document.domain, "www.web-platform.test")
```

Comment:
This might need to be a bit more involved to work in existing setups I believe...